config/keptn/org.yaml

```diff
@@ -12,7 +12,8 @@ members_can_create_repositories: false
 admins:
   # governance committee
   - AnaMMedina21
-  - AloisReitbauer
```

Comment:
alois needs to be added to the org members in order to be able to put him into the `emeriti` team

Reply:
done